app/shared/resource-map/MapContainer.js

```diff
@@ -72,6 +76,7 @@ export class UnconnectedResourceMapContainer extends React.Component {
         <Map
           center={center}
           className="app-ResourceMap__map"
+          onClick={this.handleOnClick}
```

Comment:
- [ ] Here you can just use the `this.props.actions.searchMapClick`. No need for a separate handle method. If there was a need the method should be named just `handleClick` or `handleMapClick` this case.